src/ui/components/Recordings/Recording.js

```diff
@@ -18,28 +41,46 @@ const DropdownPanel = ({ editingTitle, setEditingTitle, onDeleteRecording, recor
       <div className="menu-item" onClick={() => onDeleteRecording(recordingId)}>
         Delete Recording
       </div>
+      {!features.private ? null : isPrivate ? (
+        <div className="menu-item" onClick={toggleIsPrivate}>
+          Make recording public
```

Comment:
Generally we can assume context with copy `Make public` vs `Make recording public`